File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    // first ask user for input
    do
    {
        n = get_int("Height: ");
    }
    while (n < 0 || n > 23);

    // use input to build pyramid, line by line
    for (int i = 0; i < n; ++i)
    {
        // print required amount of spaces
        for (int j = 0; j < n - i - 1; ++j)
        {
            printf(" ");
        }
        // print block for left half of pyramid
        for (int j = 0; j < i + 2; ++j)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

# prompt for user input
while True:
    n = get_int("Height: ")
    if n >= 0 and n < 24:
        break

# print the pyramid
for i in range(n):
    print(" " * (n - i - 1), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1))